src/main/java/io/jenkins/plugins/opentelemetry/OpenTelemetryConfiguration.java

```diff
@@ -18,11 +18,18 @@
     private final String endpoint;
     private final String trustedCertificatesPem;
     private final OtlpAuthentication authentication;
+    private final int collectorTimeout;
+    private final int exportInterval;
+    private final String ignoredSteps;
```

Comment:
Should we handle as a collection of String to not have to explain what is the separator...

Reply:
Do you mean the `JCasC` should be a list of strings instead a comma-separated list?

Reply:
I'm trying to convert the comma separated string to a List of Strings. Unfortunately, JCasC and the config.jelly are not working as expected :/

Reply:
It totally makes sense to me to have a single string being a comma separated list of values on the stapler layer (`JenkinsOpenTelemetryPluginConfiguration`). On the pure object oriented layers, I would see benefits in having a collection of strings (`OpenTelemetryConfiguration`).